### PR TITLE
[Merge After Upgrade] Cleanup in preparation of new docs

### PR DESCRIPTION
**Pull request by gamarin2**
**DO NOT MERGE BEFORE HUB SUCCESSFULLY UPGRADES**

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Comment by fedekunze** [comment]
@gamarin2 is this R4R ?

File: docs/delegators/delegator-security.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+* Second,  be mindful of account recovery methods used to regain access to your most important accounts and make sure that you do not use SMS as a recovery method. If you haven't done so yet, start using an authenticator app or a hardware key immediately for your personal email account and wherever else you manage your tokens, especially if you use online exchanges.
+
+
+## Supply Chain Attacks
+Whether you're buying a hardware or a hardware wallet, it is important  to purchase whatever you need directly from the supplier or from a trusted source. This is the only way to completely eliminate the risk of a compromised device or chip from stealing your private keys, especially since there are reports of compromised wallets being sold on Amazon and through other popular online marketplaces. 
+
+## Alpha Software
+
+At launch, the official Cosmos Wallet is in alpha and is unsafe to use on the Cosmos network. The Tendermint KMS is also in alpha, and is not production-ready.
```
Comment (dogemos) [inline docs/delegators/delegator-security.md]: Is Lunie still in alpha?

Comment (fedekunze) [reply docs/delegators/delegator-security.md]: It’s not alpha

**Review by fedekunze — APPROVED** [review]
utACK

**Pull request merged**